### ISSUE TITLE
Service level missing from config in V5.0.0

### ISSUE THREAD

**Issue by glynhudson** [OpenEVSE/openevse_esp32_firmware#740]
On V5.0.0 "Service Level" option is missing from Settings > EVSE 

Is there an API call which can be used to enable Service Level 2 when the option is not shown in the GUI? RAPI is disabled so `$SL 2` does not work. Is there a workaround to temporally re-enable RAPI ? 

![Screenshot-from-2023-10-10-19-00-36](https://github.com/OpenEVSE/openevse_esp32_firmware/assets/758844/fc9cc35e-8e66-4f82-a6cd-7c35051a0644)

**Comment by mathieucarbou** [comment]
Hi guys,

We saw this issue this week with @Mathopenevse when trying to troubleshoot his issues and this was very frustrating.

Hopefully there is a workaround at the moment, thanks to the PR that i'v cent, allowing config updates through MQTT.

On can send this payload to `openevse/config/set`:

```json
{"service": 2}
```

or

```json
{"service": 1}
```

**Comment by chris1howell** [comment]
Strange. It shows up for me on 5.0.0. Did you try clearing cache?

**Comment by mathieucarbou** [comment]
yes. I have like Glen's screenshot.

![image](https://github.com/OpenEVSE/openevse_esp32_firmware/assets/61346/6b30dd9d-949c-4f6f-9dd5-dfd2e25bc894)

![image](https://github.com/OpenEVSE/openevse_esp32_firmware/assets/61346/27e860a3-a719-4c49-8229-7f651bd8e970)

**Comment by KristianDebrenti** [comment]
Hello, same for me.
![Screenshot_20231010_231539_Chrome](https://github.com/OpenEVSE/openevse_esp32_firmware/assets/48323301/dd47ab15-6681-43a5-9ab2-d737da8b968b)
![Screenshot_20231010_231410_Chrome](https://github.com/OpenEVSE/openevse_esp32_firmware/assets/48323301/c2bf42d9-5925-438f-80b5-ff5f067ac645)

**Comment by mathieucarbou** [comment]
@chris1howell : maybe the explanation is because you have `8.2.3` instead of `8.2.3.EU` like us ?

We have the European firmware so only level 2 is allowed. The problem is that for some reasons or in case of reset it can happen that the SL is switched back to 1 and we have to reset it to 2. It happened a friend twice.

Here is the code proving that ;-)

```
			{#if !$config_store.firmware.endsWith(".EU") && !$config_store.firmware.endsWith(".T2")}
			<div class="my-1 is-flex is-justify-content-center" >
				<Borders grow={true}>
					<div class="is-uppercase has-text-weight-bold is-size-6 mb-3">{$_("config.evse.service")}</div>
					<Select 
						bind:this={formdata.service.input}
						bind:value={formdata.service.val} 
						bind:status={formdata.service.status} 
						items={service_items} 
						onChange={()=>setProperty("service")}
					/>
				</Borders>
			</div>
			{/if}
```

**Comment by chris1howell** [comment]
You may be correct. I am on 8.2.2.

**Comment by jeremypoulter** [comment]
Fixed via OpenEVSE/openevse-gui-v2#15
